src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -0,0 +1,966 @@
+
+            // Update report variables
+            outletNode_Num = state.dataHVACFan->fanObjs[state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanIndex]->outletNodeNum;
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_MassFlowRate =
+                state.dataLoopNodes->Node(outletNode_Num).MassFlowRate;
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_VolumeFlowRate =
+                state.dataLoopNodes->Node(outletNode_Num).MassFlowRate /
+                state.dataEnvrn->StdRhoAir; // temporaility use Std, can calculate using T and HR by calling Pschro functions.
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_Power =
+                state.dataHVACFan->fanObjs[state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanIndex]->fanPower();
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_Energy =
+                state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_Power * state.dataHVACGlobal->TimeStepSys *
+                DataGlobalConstants::SecInHour;
+
+        } else if (state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanTypeNum == DataHVACGlobals::FanType_ComponentModel) {
+            // 2022-01: Component model fan
+            // 2022-01: use a simpler call instead (checkout SimAirServingZones.cc)
+            EnergyPlus::Fans::SimulateFanComponents(state,
+                                                    state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanName,
+                                                    FirstHVACIteration,
+                                                    state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanIndex); //,
+            // FanSpeedRatio, // ZoneCompTurnFansOn, // ZoneCompTurnFansOff);
+
+            // Update output variables
+
+            auto &fancomp = state.dataFans->Fan(state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).CentralFanIndex);
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_MassFlowRate = fancomp.OutletAirMassFlowRate;
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_VolumeFlowRate =
+                fancomp.OutletAirMassFlowRate / state.dataEnvrn->StdRhoAir; // fancomp.RhoAirStdInit;
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_Power = fancomp.FanPower * 1000.0;
+
+            state.dataZoneEquip->ExhaustAirSystem(ExhaustAirSystemNum).centralFan_Energy = fancomp.FanEnergy * 1000.0;
+        }
+
+        // 2022-01: Determine if there are some "iteration" or revisit step for the zone mixer and fan simulation
+        Real64 mixerFlow_Posterior = 0.0;
+        mixerFlow_Posterior = state.dataLoopNodes->Node(outletNode_index).MassFlowRate;
+        if (mixerFlow_Posterior < 1e-6) {
```

Comment:
1e-6 should be a `constexpr Real64` somewhere.

Reply:
Found one DataHVACGlobals::SmallMassFlow (1e-3) to use.